packages/cli/src/commands/init.ts

```diff
@@ -17,23 +20,39 @@ export default class Init extends Command {
   static args = [
     {
       name: 'projectName',
-      required: true,
       description: 'Give the starter project name',
     },
   ];
 
   async run(): Promise<void> {
     const {args, flags} = this.parse(Init);
-    if (flags.starter && args.projectName) {
-      this.log('Init the starter package');
+    const project = {} as projectSpec;
+
+    project.name = args.projectName
+      ? args.projectName
+      : ((await cli.prompt('What is the project name?', {default: 'subql-starter', required: true})) as string);
+    if (fs.existsSync(path.join(process.cwd(), `${project.name}`))) {
+      throw new Error(`Directory ${project.name} exists, try another project name`);
+    }
+    project.repository = (await cli.prompt('What is the project git repository?', {required: false})) as string;
+    project.endpoint = (await cli.prompt('What is the RPC endpoint?', {
+      default: 'wss://polkadot.api.onfinality.io/public-ws',
+      required: true,
+    })) as string;
+    project.author = (await cli.prompt('What is the authors name?', {required: true})) as string;
```

Comment:
Need to unify the prompt styles. Sometimes you use question and sometimes simply the item name.